requirements.txt

```diff
@@ -1,5 +1,5 @@
 blessings
 future>=0.14.0
-gherkin-official
+gherkin-official>=4
```

Comment:
Should we pin the major version a bit more tightly to prevent everything from melting?